test/unit/spec/common/jobStateViewer-spec.js

```diff
@@ -0,0 +1,348 @@
+define([
+    'jquery',
+    'bluebird',
+    'common/jobStateViewer',
+    'common/jobs',
+    'common/jobCommMessages',
+    'common/jobManager',
+    'common/props',
+    'common/runtime',
+    '/test/data/jobsData',
+    'testUtil',
+], (
+    $,
+    Promise,
+    JobStateViewerModule,
+    Jobs,
+    jcm,
+    JobManagerModule,
+    Props,
+    Runtime,
+    JobsData,
+    TestUtil
+) => {
+    'use strict';
+
+    const { cssBaseClass, ManagedJobStateViewer } = JobStateViewerModule;
```

Comment:
Created this test module by picking out the relevant tests from the job log viewer unit tests.